### PR TITLE
Adding page for masterclass links

### PR DESCRIPTION
**Pull request by garethtdavies**
Change-type: minor
Signed-off-by: Gareth Davies gareth@balena.io

### CODE REVIEW

File: pages/learn/more/masterclass.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+---
+title: Masterclasses
+---
+
+# {{ $names.company.lower }} Masterclasses
+
+Masterclasses are a series of self-contained, in-depth exercises designed to take you from a {{ $names.company.lower }} beginner to power user. Each masterclass focusses on providing in-depth knowledge on a specific part of the {{ $names.company.lower }} system.
```
Comment (xginn8) [inline pages/learn/more/masterclass.md]:
```suggestion
Masterclasses are a series of self-contained, in-depth exercises designed to take you from a {{ $names.company.lower }} beginner to power user. Each masterclass focuses on providing in-depth knowledge on a specific part of the {{ $names.company.lower }} system.
```

File: pages/learn/more/masterclass.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+Feedback on the masterclasses are welcome and should be given by creating an issue in the respective masterclass repository.
+
+Currently, the following masterclasses are available:
+
+* [{{ $names.cli }} masterclass]({{ $links.githubProjects }}/balena-cli-masterclass) - a guide to getting started with the {{ $names.company.lower }}.
```
Comment (xginn8) [inline pages/learn/more/masterclass.md]:
```suggestion
* [{{ $names.cli }} masterclass]({{ $links.githubProjects }}/balena-cli-masterclass) - a guide to getting started with the {{ $names.cli }}.
```

File: pages/learn/more/masterclass.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
+---
+title: Masterclasses
+---
+
+# {{ $names.company.lower }} Masterclasses
```
Comment (xginn8) [inline pages/learn/more/masterclass.md]:
```suggestion
# {{ $names.company.upper }} Masterclasses
```

File: pages/learn/more/masterclass.md (hunk 1 above)
Comment (garethtdavies) [reply pages/learn/more/masterclass.md]: Thanks, going to take me a while to get used to those spellings 😄

**Comment by garethtdavies** [comment]
@balena-ci retest

**Comment by xginn8** [comment]
@garethtdavies just dropping this bash snippet to grab all repos based on a topic, then grab their descriptions:
```
for i in $(curl -H 'Accept: application/vnd.github.mercy-preview+json' 'https://api.github.com/search/topics?q=balena-masterclass' | jq -r ".[]" | jq -r ".[].name"); do curl "https://api.github.com/repos/balena-io-projects/$i" | jq .description; done
```

if we apply that topic to all repos, we can generate this snippet pretty easily going forward. I would _assume_ we can extend the template used (https://github.com/balena-io-projects/balena-masterclass-skeleton) to prefill that topic as well, eliminating another manual step from the discovery process. just food for thought!

**Comment by hedss** [comment]
@garethtdavies Heya! Given the approvals, can we get this rebased and merged please? :)

**Comment by garethtdavies** [comment]
@balena-ci retest

**Comment by garethtdavies** [comment]
@hedss Done. Wasn't sure if we were waiting on any more before pulling the switch but very happy to get this out there.

**Pull request merged**